src/hotspot/share/opto/loopnode.cpp

```diff
@@ -5232,6 +5253,68 @@ int PhaseIdealLoop::build_loop_tree_impl( Node *n, int pre_order ) {
   return pre_order;
 }
 
+#ifndef PRODUCT
```

Comment:
Should be `#ifdef ASSERT` because it is called in `DEBUG_ONLY()` macro.

Reply:
:+1: